.travis.yml

```diff
@@ -18,9 +19,9 @@ before_script:
   - sleep 5
 script:
   - npm run lint
-  - npm run unit
+  - if [[ $(node -v) =~ ^v4.* ]]; then npm run unit:harmony; else npm run unit; fi
```

Comment:
That sounds good, except for the && and ||. I've personally refrained from using them because if npm run unit:harmony fails then npm run unit will be run, which is not what we want.